### PR TITLE
GUACAMOLE-1245: Add client parameters for specifying WoL port.

### PR DESCRIPTION
**Pull request by necouchman**

### CODE REVIEW

File: guacamole/src/main/webapp/translations/fr.json [hunk 1]
```diff
@@ -519,10 +519,10 @@
...
         "FIELD_HEADER_USERNAME"        : "Identifiant:",
         "FIELD_HEADER_WIDTH"           : "Largeur:",
-      "FIELD_HEADER_WOL_BROADCAST_ADDR" : "Adresse de diffusion pour les paquets WoL:",
-      "FIELD_HEADER_WOL_MAC_ADDR"       : "Adresse MAC de l'hôte distant:",
-      "FIELD_HEADER_WOL_SEND_PACKET"    : "Envoi de paquets WoL:",
-      "FIELD_HEADER_WOL_WAIT_TIME"      : "Temps d'attente du démarage de l'hôte:",
+        "FIELD_HEADER_WOL_BROADCAST_ADDR" : "Adresse de diffusion pour les paquets WoL:",
+        "FIELD_HEADER_WOL_MAC_ADDR"       : "Adresse MAC de l'hôte distant:",
+        "FIELD_HEADER_WOL_SEND_PACKET"    : "Envoi de paquets WoL:",
+        "FIELD_HEADER_WOL_WAIT_TIME"      : "Temps d'attente du démarage de l'hôte:",
```
Comment (mike-jumper) [inline guacamole/src/main/webapp/translations/fr.json:525]: This doesn't look to be related to GUACAMOLE-1245. Was this intended to be part of this PR?

Comment (necouchman) [reply guacamole/src/main/webapp/translations/fr.json]: I was just cleaning up the spacing while I was in there, but I've pulled it out of this PR.

**Review by mike-jumper — APPROVED** [review]
